contracts/payloads/IGP102/PayloadIGP102.sol

```diff
@@ -150,51 +153,84 @@ contract PayloadIGP102 is PayloadIGPMain {
 
     // @notice Action 4: Collect stETH, wstETH revenue and deposit from reserve contract to Lite Vault
     function action4() internal isActionSkippable(4) {
-        address[] memory tokens = new address[](2);
+        {
+            address[] memory tokens = new address[](2);
 
-        tokens[0] = stETH_ADDRESS;
-        tokens[1] = wstETH_ADDRESS;
+            tokens[0] = ETH_ADDRESS;
+            tokens[1] = wstETH_ADDRESS;
 
-        LIQUIDITY.collectRevenue(tokens);
+            LIQUIDITY.collectRevenue(tokens);
 
-        // Deposit stETH, wstETH into Lite Vault
-        string[] memory targets = new string[](2);
-        bytes[] memory encodedSpells = new bytes[](2);
+            uint256[] memory amounts = new uint256[](2);
 
-        string
-            memory depositSignature = "deposit(address,uint256,uint256,uint256)";
+            amounts[0] = address(FLUID_RESERVE).balance - 0.1;
+            amounts[1] =
+                IERC20(wstETH_ADDRESS).balanceOf(address(FLUID_RESERVE)) -
+                0.1;
 
-        // Spell 1: Deposit stETH into Lite
+            FLUID_RESERVE.withdrawFunds(tokens, amounts, TREASURY); // Withdraw to Treasury
```

Comment:
withdraw we also need to withdraw stETH which ended up on the reserve contract some time ago through other ways